artemis-core-client/src/main/java/org/apache/activemq/artemis/core/message/impl/CoreMessage.java

```diff
@@ -106,11 +106,12 @@
    private final CoreMessageObjectPools coreMessageObjectPools;
 
    public CoreMessage(final CoreMessageObjectPools coreMessageObjectPools) {
+      new Exception("new coreMessage").printStackTrace();
```

Comment:
?

Reply:
geez.. removing it

Reply:
that's a debug trick that leaked on the PR :)